frontend/device/devicelistener.lua

```diff
@@ -207,6 +207,19 @@ if Device:hasGSensor() then
         return true
     end
 
+    function DeviceListener:onTempGSensorOn()
+        if not G_reader_settings:isTrue("input_ignore_gsensor") then
+            Notification:notify("Accelerometer rotation events already on.")
+        else
+            Device:toggleGSensor()
```

Comment:
shouldn't that be toggleGSensor(true) and then the scheduled one toggleGSensor(false)?

Reply:
I just tested this but it only worked to turn it on, but then it didn't turn off when it should.